spec/controllers/grade_entry_forms_controller_spec.rb

```diff
@@ -149,12 +149,13 @@
     end
 
     it 'ignores the total column if given in the csv file' do
+      # show_total needs to be on when uploading a csv file with the "Total" column included
```

Comment:
Rather than including this in a comment, why don't you change the description on the previous line to specify a "when ...".